main.go

```diff
@@ -1,7 +1,6 @@
 package main
 
 import (
-	_ "net/http/pprof"
```

Comment:
Don't we need to run init in pprof? I thought this was a purpose, so it was added here.

Reply:
nice catch...
linter suggest that I delete it at first.

I just tested the profiler again, and yes, we need this for the profiler.
I added this back and added the nolint and comment for others.